posthog/api/organization.py

```diff
@@ -215,28 +215,24 @@ def dangerously_get_permissions(self):
             ]
             if not is_cloud():
                 create_permissions.append(PremiumMultiorganizationPermission())
+
             return create_permissions
 
-        if self.action == "update":
-            create_permissions = [
+        if self.action in ["update", "partial_update"]:
+            update_permissions = [
                 permission()
-                for permission in [permissions.IsAuthenticated, TimeSensitiveActionPermission, APIScopePermission]
-            ]
-
-            if any(
```

Comment:
This opens up risk for editing other columns as a member and I don't think there are any needed for that. If we find there are we can flip it and do a whitelist.